src/Web/Sfa.Das.Sas.Web/Views/Provider/ProviderDetail.cshtml

```diff
@@ -27,8 +27,55 @@
                     <dd id="trading-names">@Model.TradingNames</dd>
                 }
                 <dt>UKPRN:</dt>
-                <dd>@Model.UkPrn</dd>
+                <dd>@Model.UkPrn</dd>                   
             </dl>
+            <p class="markdown-header-text">About this Provider</p>
+            @if (string.IsNullOrEmpty(Model.MarketingInfo))
+            {
+                <span>There is no marketing information available about this provider.</span>
+            }
+            else
+            {
+                @Html.MarkdownToHtml(Model.MarketingInfo)
+            }
+            
+  
+            @if (Model.ApprenticeshipTrainingSummary != null && Model.ApprenticeshipTrainingSummary.Count > 0)
+            {
+               <div id="search-app-offered">
+                     <h2 class="heading-medium app-offered">
+                        @Model.ApprenticeshipTrainingSummary.Count Apprenticeships offered
+                    </h2>
+                    <!--search box-->               
+                    @*<form id="search-box" method="get" action="#">
+                            <fieldset class="inline">
```

Comment:
Commented code

Reply:
Yeah, Rajesh put that in as it will be needed in a future story. I'll take it out but keep a copy